app/src/admin/users.tsx

```diff
@@ -248,6 +252,7 @@ function EnhancedTableHead(props: EnhancedTableProps) {
     );
 }
 
+//Main table displayed on page
 const MainFrame = (props: MainframeProps) => {
     const [order, setOrder] = useState<Order>('asc');
```

Comment:
so instead of doing `if (!props.rows.data)`, you can just set a default when you declare the props here

```suggestion
const MainFrame: React.FC<MainframeProps> = ({
  handleRoleFilterOpen, 
  handleRoleUpdate, 
  rows = []
}) => {
    const [order, setOrder] = useState<Order>('asc');
```